spring-petclinic-microservices/pom.xml

```diff
@@ -3,228 +3,228 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.3.6.RELEASE</version>
```

Comment:
It's better to use the same version of spring-boot-starter-parent

Reply:
I agree, but we need to test the big sample, not just upgrade spring-boot-starter-parent.